src/assets/config/appConfig.json.template

```diff
@@ -1,3 +1,4 @@
 {
-    "NEWS_URL": "%NEWS_URL%"
+  "NEWS_URL": "%NEWS_URL%",
+  "HUB_URL": "%HUB_URL%"
```

Comment:
This will need to be added to line 92 of the openshift/templates/runtime/s2i/bin/run file